main.js

```diff
@@ -5,6 +5,7 @@ const constructAll = function() {
 	document.removeEventListener('o.DOMContentLoaded', constructAll);
 };
 
-document.addEventListener('o.DOMContentLoaded', constructAll);
-
+if (typeof window !== 'undefined') {
```

Comment:
They would, I'm wondering why this `if` statement exists, what are you protecting against? Running in a web worker?